skillmap/src/App.tsx

```diff
@@ -361,6 +361,19 @@ class AppImpl extends React.Component<AppProps, AppState> {
         }
     }
 
+    componentDidUpdate() {
+        const { highContrast } = this.props;
+
+        const bodyIsHighContrast = document.body.classList.contains("high-contrast");
```

Comment:
can we remove the high-contrast class from the app-container element with this change?

Reply:
done